nixos/tests/k3s/default.nix

```diff
@@ -1,9 +1,18 @@
 { system ? builtins.currentSystem
 , pkgs ? import ../../.. { inherit system; }
 }:
-{
-  # Run a single node k3s cluster and verify a pod can run
-  single-node = import ./single-node.nix { inherit system pkgs; };
-  # Run a multi-node k3s cluster and verify pod networking works across nodes
-  multi-node = import ./multi-node.nix { inherit system pkgs; };
-}
+let
+  allK3s = with pkgs; [
+    k3s_1_23
+    k3s_1_24
+    k3s_1_25
+    k3s_1_26
+  ];
```

Comment:
```suggestion
  allK3s = {
    (pkgs)
      k3s
      k3s_1_23
      k3s_1_24
      k3s_1_25
      k3s_1_26
    ;
  };
```

Reply:
No `inherit`?

Reply:
Whoops, fixed it. @yajo should ensure it works on their own anyways, this is just a suggestion via GH's limited system. I did not test this.

Reply:
Was just clarifying. No worry.

Reply:
This changed the structure from a attrlist of tests to an attrlist of attrlists of tests. I guess it doesn't matter.

Reply:
Yeah, that's intentional ;)